pkg/runner/run_context.go

```diff
@@ -202,8 +203,10 @@ func (rc *RunContext) startJobContainer() common.Executor {
 			rc.JobContainer.Start(false),
 			rc.JobContainer.UpdateFromImageEnv(&rc.Env),
 			rc.JobContainer.UpdateFromEnv("/etc/environment", &rc.Env),
-			rc.JobContainer.Exec([]string{"mkdir", "-m", "0777", "-p", ActPath}, rc.Env, "root", ""),
+			rc.JobContainer.Exec([]string{"mkdir", "-m", "0777", "-p", ActPath}, rc.Env, "0", ""),
 			rc.JobContainer.CopyDir(copyToPath, rc.Config.Workdir+string(filepath.Separator)+".", rc.Config.UseGitIgnore).IfBool(copyWorkspace),
+			rc.JobContainer.Exec([]string{"chown", "-R", rc.Config.ContainerUser, copyToPath}, rc.Env, "root", "").IfBool(copyWorkspace),
+			rc.JobContainer.Exec([]string{"chown", "-R", rc.Config.ContainerUser, "/home/runner"}, rc.Env, "root", ""),
```

Comment:
This will fail on image that don't have such path

Reply:
Which images would those be?  I thought the runner software was always installed in act images.

Reply:
All images except `large` will not have that path

Reply:
I've been tinkering with some of the other images in the README, and I see what you mean.  I'm going to experiment first with creating that folder and setting the value of the `HOME` environment variable to that.

What I've found so far is that if the UID specified in --container-user doesn't exist in the container, `$HOME` ends up being `/`, which breaks a bunch of my workflows.  It should be an easy fix to create and set a home directory, though.

Let me know if you think this is the wrong direction.

Reply:
I don't think it's inherently wrong but it's easy to get it wrong.

We used to set `HOME` in act and that broke things. Best option would be to look for home path in `/etc/passwd`

Reply:
In the case I'm describing, though, there is no corresponding /etc/passwd entry.

Reply:
Yes, that's correct, I misread that